core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -1035,6 +1031,30 @@ public void modifyProperties(String tableName, final Consumer<Map<String,String>
     }
   }
 
+  @Override
+  public void modifyProperties(String tableName, final Consumer<Map<String,String>> mapMutator)
+      throws AccumuloException, AccumuloSecurityException, IllegalArgumentException {
+    EXISTING_TABLE_NAME.validate(tableName);
+    checkArgument(mapMutator != null, "mapMutator is null");
+
+    Retry retry =
+        Retry.builder().infiniteRetries().retryAfter(25, MILLISECONDS).incrementBy(25, MILLISECONDS)
+            .maxWait(30, SECONDS).backOffFactor(1.5).logInterval(3, MINUTES).createRetry();
+
```

Comment:
Is it correct to have the log interval greater than max wait? Does this efficiently eliminate logging except on timeout?

Reply:
Good question.  I copied and pasted that code and modified it a bit. I went a looked at the code and logging does not happen unless an explicit method call is made.  I need to look into this some more, see if I should call the logging method or remove the config from the builder.

Reply:
I looked more closely at the retry code and made some updates in 3873e94.

Below are some log messages I saw while running the new IT added in this PR.

```
2022-09-28T14:31:47,217 [admin.TableOperations] DEBUG: Unable to modify table properties for PropStoreConfigIT_concurrentTablePropsModificationTest0 because of concurrent modification, retrying attempt 1 (suppressing retry messages for 180000ms)
2022-09-28T14:31:47,217 [util.Retry] DEBUG: Sleeping for 25ms before retrying operation
2022-09-28T14:31:47,252 [admin.TableOperations] DEBUG: Unable to modify table properties for PropStoreConfigIT_concurrentTablePropsModificationTest0 because of concurrent modification, retrying attempt 1 (suppressing retry messages for 180000ms)
2022-09-28T14:31:47,252 [util.Retry] DEBUG: Sleeping for 25ms before retrying operation
2022-09-28T14:31:47,283 [util.Retry] DEBUG: Sleeping for 62ms before retrying operation
2022-09-28T14:31:47,330 [util.Retry] DEBUG: Sleeping for 150ms before retrying operation
2022-09-28T14:31:47,332 [util.Retry] DEBUG: Sleeping for 158ms before retrying operation
2022-09-28T14:31:47,362 [util.Retry] DEBUG: Sleeping for 79ms before retrying operation
2022-09-28T14:31:47,458 [util.Retry] DEBUG: Sleeping for 111ms before retrying operation
2022-09-28T14:31:47,497 [util.Retry] DEBUG: Sleeping for 217ms before retrying operation
2022-09-28T14:31:47,500 [util.Retry] DEBUG: Sleeping for 215ms before retrying operation
2022-09-28T14:31:47,578 [util.Retry] DEBUG: Sleeping for 153ms before retrying operation
2022-09-28T14:31:47,727 [util.Retry] DEBUG: Sleeping for 302ms before retrying operation
2022-09-28T14:31:47,734 [admin.TableOperations] DEBUG: Unable to modify table properties for PropStoreConfigIT_concurrentTablePropsModificationTest0 because of concurrent modification, retrying attempt 1 (suppressing retry messages for 180000ms)
2022-09-28T14:31:47,735 [util.Retry] DEBUG: Sleeping for 25ms before retrying operation
2022-09-28T14:31:47,746 [admin.TableOperations] DEBUG: Unable to modify table properties for PropStoreConfigIT_concurrentTablePropsModificationTest0 because of concurrent modification, retrying attempt 1 (suppressing retry messages for 180000ms)
2022-09-28T14:31:47,746 [util.Retry] DEBUG: Sleeping for 25ms before retrying operation
```